droidlet/tools/hitl/dashboard_app/backend/dashboard_aws_helper.py

```diff
@@ -59,32 +66,6 @@ def _read_file(fname: str):
     return content
 
 
-def _dowload_file(fname: str):
-    # check if exists on local tmp directory
-    local_file_name = os.path.join(HITL_TMP_DIR, fname)
-
-    if not os.path.exists(local_file_name):
-        # reterive from s3
-        local_folder_name = local_file_name[: local_file_name.rindex("/")]
-        os.makedirs(local_folder_name, exist_ok=True)
-        try:
-            s3.meta.client.download_file(S3_BUCKET_NAME, fname, local_file_name)
-        except botocore.exceptions.ClientError as e:
-            print(f"file not exists {fname}")
-
-    return local_file_name if os.path.exists(local_file_name) else None
-
-
-def _read_file(fname: str):
-    """
-    read file into a string
-    """
-    f = open(fname, "r")
-    content = f.read()
-    f.close()
-    return content
-
-
```

Comment:
The `_dowload_file`, `_read_file` and `get_traceback_by_id` are duplicated function definitions due to rebase before.